plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMinioParquetConnectorSmokeTest.java

```diff
@@ -31,4 +32,16 @@ protected boolean isFileSorted(Location path, String sortColumnName)
     {
         return checkParquetFileSorting(fileSystem.newInputFile(path), sortColumnName);
     }
+
+    @Test
+    public void testQueryHadoopFsTableInNativeFsWithS3anScheme()
+    {
+        String bucket = "trino-ci-test";
+        String tableName = "sample_table1";
+        hiveMinioDataLake.getMinio().createBucket(bucket);
+        hiveMinioDataLake.getMinio().copyResources("iceberg/hadoopFss3anTesting/level1/level2/sample_table1/", bucket, "level1/level2/sample_table1/");
+        assertQueryFails("CALL system.register_table (CURRENT_SCHEMA, '" + tableName + "', 's3a://trino-ci-test/level1/level2/sample_table1')", "Table metadata file \\[s3a://trino-ci-test/level1/level2/sample_table1/metadata/00002-8eae2776-7089-4dbd-8d1b-c2b96e75ab80.metadata.json] declares table location as \\[s3a://trino-ci-test/level1/level2//sample_table1] which is differs from location provided \\[s3a://trino-ci-test/level1/level2/sample_table1]. Iceberg table can only be registered with the same location it was created with.");
```

Comment:
It is not clear from my perspective what we're testing here.
Advocating for removal

Reply:
It was for reference that copying to double slash doesn't work with MinIO. I was about to remove it